WireDomain/Sources/WireDomain/Repositories/Conversations/ConversationRepository.swift

```diff
@@ -185,4 +226,48 @@ public final class ConversationRepository: ConversationRepositoryProtocol {
         )
     }
 
+    public func deleteMLSConversation(
+        with id: UUID,
+        domain: String?
+    ) async throws {
+        guard let conversation = await fetchConversation(
+            with: id,
+            domain: domain
+        ) else {
+            return WireLogger.conversation.warn(
+                "Cannot delete a MLS conversation that doesn't exist locally: \(id.safeForLoggingDescription)"
+            )
+        }
+
+        guard let mlsGroupID = conversation.mlsGroupID else {
+            throw ConversationRepositoryError.mlsConversationShouldHaveAGroupID
+        }
+
+        try await conversationsLocalStore.wipeMLSGroup(with: mlsGroupID)
```

Comment:
Shouldn't this be

```suggestion
        if let mlsGroupID = conversation.mlsGroupID {
            try await conversationsLocalStore.wipeMLSGroup(with: mlsGroupID)
        }
```

Reply:
yes, you're right, thx.
fixed in https://github.com/wireapp/wire-ios/pull/2034/commits/f69fec906f765270a691be84d560edd387cc9010